pgmpl/figure.py

```diff
@@ -196,10 +201,14 @@ def gca(self):
         Imitation of matplotlib gca()
         :return: Current axes for this figure, creating them if necessary
         """
+        if self.axes is not None:
+            try:
+                ax = list(flatten(np.atleast_1d(self.axes)))[-1]
+            except RuntimeError:  # Happens if Qt has deleted the Axes
+                print('Warning: Qt has deleted the axes; figure had a residual reference to a bad Qt object. (gca)')
+                self.axes = None
```

Comment:
I think codecov is complaining about these three lines not executing in the test. Ugh. This is a weird error to begin with, and it IS being hit earlier (lines 140-142) where I have to try/except to catch the same thing. I'm going to just accept this for now. Maybe I can figure out what exactly causes this and how to reliably reproduce it to hit these three, too. As it is, I can't even hit these errors by running tests individually; it's some combination that does this, I guess. 

So, proceeding against codecov complaint.